package.json

```
@@ -1,24 +1,110 @@
 {
-  "name": "go-test",
-  "version": "1.0.0",
-  "description": "",
+  "name": "go-template",
+  "version": "0.0.1",
+  "description": "Template for generating Go module based on async api",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"no test specified yet\"",
-    "lint": "echo \"no lint configured yet\"",
-    "generate:assets": "echo \"no assets to generate yet\"",
-    "develop": "ag test/asyncapi.yaml ./ -o test/output --force-write --watch-template"
+    "test": "npm run testTemplate && npm run testHooks && npm run testFilters",
+    "testTemplate": "rimraf test/temp && jest --coverage -t 'templateGenerationResult'",
+    "testHooks": "rimraf test/temp && jest --coverage -t 'generateExtraFormats'",
+    "testFilters": "rimraf test/temp && jest --coverage -t 'generateMermaidDiagram'",
+    "release": "echo 'We do not trigger releases for this template. Normally this script should invoke \"semantic-release\"'",
```

this time we want to release it

```suggestion
    "release": "semantic-release",
```